src/strings/en-us.json

```diff
@@ -971,6 +971,8 @@
     "LabelTypeText": "Text",
     "LabelUnstable": "Unstable",
     "LabelUser": "User",
+    "LabelUseReplayGainTagsHelp": "Scan audio files for replaygain tags and use them instead of computing LUFS value. (Uses less computing power. Will override 'LUFS Scan' option)",
```

Comment:
I found this wording a little confusing... does it mean that if replaygain tags are found then the LUFS scan will be skipped for _that_ file?

Reply:
Yeah that is correct